symplyphysics/laws/kinematic/damped_oscillations/underdamping.py

```diff
@@ -0,0 +1,79 @@
+from sympy import Eq, exp, cos
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+from symplyphysics.core.symbols.quantities import scale_factor
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.definitions import damped_harmonic_oscillator_equation as damped_eqn
+
+# Description
+## In the presence of a damping force in the oscillating system, the system's behaviour
+## depends on the value of the damping ratio. When it is less than 1, the system oscillates
+## with a slightly different frequency than in the undamped case, and its amplitude decreasing
+## to zero. This behaviour is also known as underdamping.
+
+# Law: x(t) = x_max * exp(-lambda*t) * cos(omega_damped*t + phi)
+## x(t) - position of damped oscillator
+## x_max - amplitude of damped oscillations
```

Comment:
But amplitude changes over time. Maybe it should be initial amplitude or initial position?

Reply:
Hm, in reality it's none of those, but just some constant $`A`$ that needs to be found from the boundary equation $`x(0) = A \cos(\phi)`$. It would be the initial amplitude if $\phi = 0$.

Reply:
Yes, but position is never higher than `A` for underdamped oscillator. So it is maximum amplitude.

Reply:
It is a non-strict upper boundary of $`x`$ but that doesn't mean it ever reaches that value (in the general case). Maybe we can leave it for the end user to solve